src/main/java/no/nav/veilarbvedtaksstotte/client/dokarkiv/SafClientImpl.java

```diff
@@ -52,7 +54,7 @@ public byte[] hentVedtakPdf(String journalpostId, String dokumentInfoId) {
 
     @SneakyThrows
     public List<Journalpost> hentJournalposter(Fnr fnr) {
-        GraphqlRequest graphqlRequest = new GraphqlRequest(createDokumentoversiktBrukerGqlStr(fnr));
+        GraphqlRequest<DokumentOversiktBrukerVariables> graphqlRequest = new GraphqlRequest<>(createDokumentoversiktBrukerGqlStr(), new DokumentOversiktBrukerVariables(new BrukerId(fnr.get(), BrukerIdType.FNR), "OPP", 50));
```

Comment:
Extract OPP to const or first 50...

Reply:
Also check what kind of documents we get with tema OPP

Reply:
Other applications use tema OPP, but we are filtering those documents with those functions:

```
protected List<ArkivertVedtak> hentArkiverteVedtakFraArena(Fnr fnr) {
        return safClient.hentJournalposter(fnr)
                .stream()
                .filter(this::erVedtakFraArena)
                .map(this::tilArkivertVedtak)
                .filter(this::harDokumentInfoId)
                .collect(Collectors.toList());
    }
```

```
private boolean erVedtakFraArena(Journalpost journalpost) {
        return JOURNALPOST_ARENA_VEDTAK_TITTEL.equals(journalpost.tittel);
}
```